ci-operator/step-registry/aws/provision/vpc/shared/aws-provision-vpc-shared-commands.sh

```diff
@@ -570,11 +574,12 @@ fi
 
 if [[ ${CREATE_AWS_OUTPOST_SUBNETS} == "yes" ]]; then
   outpost_arn=$(jq -r '.OutpostArn' ${CLUSTER_PROFILE_DIR}/aws_outpost_info.json)
-  outpost_zone=$(jq -r '.AvailabilityZone' ${CLUSTER_PROFILE_DIR}/aws_outpost_info.json)
+  outpost_az=$(jq -r '.AvailabilityZone' ${CLUSTER_PROFILE_DIR}/aws_outpost_info.json)
   aws_add_param_to_json "CidrCount" "8" "$vpc_params"
   aws_add_param_to_json "OutpostArn" "${outpost_arn}" "$vpc_params"
-  aws_add_param_to_json "AllowedAvailabilityZoneList" "${outpost_zone}" "$vpc_params"
-  aws_add_param_to_json "AvailabilityZoneCount" "1" "$vpc_params"
+  aws_add_param_to_json "OutpostAz" "${outpost_az}" "$vpc_params"
+  aws_add_param_to_json "AllowedAvailabilityZoneList" "us-east-1a,us-east-1c" "$vpc_params"
```

Comment:
Just be curious, can we use `us-east-1b` here beside "us-east-1a,us-east-1c"

Reply:
ALB requires two different AZs, so this is for ALB tests only.

Reply:
good to know, so that means, we also can set it `us-east-1a,us-east-1b`, right?

Reply:
Yes, it can be any two AZs in `aws ec2 --region=$REGION describe-availability-zones --filter Name=zone-type,Values=availability-zone | jq -r '.AvailabilityZones[] | .ZoneName'`

Reply:
if that, we even do not need to specify `AllowedAvailabilityZoneList`, per my understanding, this parameter is only for OCP cluster install, did not involve outpost worker nodes yet, right?